### ISSUE TITLE
+ git ls-remote --exit-code --heads git@github.com:DrTtnk/examples.git main

### ISSUE THREAD

**Issue by manuel-delverme** [ministry-of-silly-code/experiment_buddy#70]
+ align_example_branch
+ cd examples
+ git ls-remote --exit-code --heads git@github.com:DrTtnk/examples.git main
+ git rev-parse --verify main
+ sed -i -e 's|\.git.*\#egg|\.git\@main\#egg|' requirements.txt
+ update_repo
+ sleep 1
+ meta exec 'git add -A           && git commit -q -m "Testing changes"           && git push -q --force           && git reset --soft HEAD~1           || echo nothing to commit'

/home/esac/projects/meta-buddy/buddy_metarepo:
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
nothing to commit
/home/esac/projects/meta-buddy/buddy_metarepo ✓

experiment_buddy:
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
nothing to commit
experiment_buddy ✓

examples:
On branch main
Your branch is ahead of 'origin/main' by 1 commit.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
nothing to commit
examples ✓
+ '[' local = local ']'
+ export DEPLOY_DESTINATION=local
+ DEPLOY_DESTINATION=local
+ run_local_deploy
+ pip -q install -r ./examples/requirements.txt
+ python ./examples/mnist_classifier.py
INFO:root:Name your run in the pop-up window!
experiment_id: buddy_test
http://localhost:6006
wandb: Currently logged in as: delvermm (use `wandb login --relogin` to force relogin)
wandb: wandb version 0.10.31 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.10.29
wandb: Syncing run buddy_test_Jun06_01-22-35
wandb: ⭐️ View project at https://wandb.ai/delvermm/buddy_metarepo
wandb: 🚀 View run at https://wandb.ai/delvermm/buddy_metarepo/runs/qxg1ljn2
wandb: Run data is saved locally in /home/esac/projects/meta-buddy/buddy_metarepo/wandb/run-20210606_012247-qxg1ljn2
wandb: Run `wandb offline` to turn off syncing.

setting ^initial_lr=0.0001
setting ^decay_steps=500000
setting ^num_hidden=1024
setting ^decay_factor=0.5
setting ^batch_size=128
setting ^momentum_mass=0.99
setting ^weight_norm=0.0
setting ^num_epochs=1000
setting ^_ignored=[]
WARNING:absl:No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
downloaded https://storage.googleapis.com/cvdf-datasets/mnist/train-images-idx3-ubyte.gz to /tmp/jax_example_data/
downloaded https://storage.googleapis.com/cvdf-datasets/mnist/train-labels-idx1-ubyte.gz to /tmp/jax_example_data/
downloaded https://storage.googleapis.com/cvdf-datasets/mnist/t10k-images-idx3-ubyte.gz to /tmp/jax_example_data/
downloaded https://storage.googleapis.com/cvdf-datasets/mnist/t10k-labels-idx1-ubyte.gz to /tmp/jax_example_data/

Starting training...

**Comment by DrTtnk** [comment]
Git remotes updated
